src/terminal/terminal.c

```diff
@@ -504,6 +505,13 @@ guac_terminal* guac_terminal_create(guac_client* client,
     /* Configure backspace */
     term->backspace = options->backspace;
 
+    /* Configure the family of codes for function keys and the keypad */
+    if (options->func_keys_and_keypad == NULL || options->func_keys_and_keypad[0] == 0) {
+        term->func_keys_and_keypad = GUAC_TERMINAL_FUNC_KEYS_AND_KEYPAD_DEFAULT;
+    } else if (strcmp(options->func_keys_and_keypad, "vt100plus") == 0) {
```

Comment:
Any particular reason why `vt100plus` instead of `vt100+` or `vt100`? The value `vt100` would be in line with the familiar value accepted by `TERM`.